docs/resources.rst

```diff
@@ -24,6 +24,9 @@ Integrated (Ethereum) Development Environments
 
     * `Dapp <https://dapp.tools/>`_
         Tool for building, testing and deploying smart contracts from the command line.
+    
+    * `Foundry <https://github.com/gakonst/foundry>`_
```

Comment:
The list is alphabetical, so if we want to add it, we should move it past embark.